src/Aura/Languages.hs

```diff
@@ -1630,7 +1634,7 @@ saveState_1 :: Language -> String
 saveState_1 = \case
     Japanese   -> "現在パッケージ状態保存完了。"
     Croatian   -> "Stanje paketa spremljeno."
-    German     -> "Paketzustände gesichert."
+    German     -> "Paketzustand gesichert."
```

Comment:
The old translation was for 'Saved package states.' instead of 'Saved package state.'.